README.md

```diff
@@ -29,16 +29,16 @@ We want this lib always up-to-date. So the API version is always the newest, in
 ## Installing
 You can install the library from following source:
 
-The latest release is always available on [NuGet](https://nuget.dspng.aitsys.dev).
+The latest release is always available on [NuGet](https://nuget.dcs.aitsys.dev).
 
 ## Documentation
-The documentation for the latest stable version is available at [docs.dspng.aitsys.dev](https://docs.dspng.aitsys.dev).
+The documentation for the latest stable version is available at [docs.dcs.aitsys.dev](https://docs.dcs.aitsys.dev).
```

Comment:
Broken url. May want to update DNS records

Reply:
yeah I'm doing the update on merge